src/test/kotlin/study/Calculator.kt

```diff
@@ -0,0 +1,43 @@
+package study
+
+class Calculator(val input: String?) {
+    fun calculate(): Double {
+        val inputValues = input?.split(" ") ?: listOf()
```

Comment:
elvis 연산자를 활용한다면 null 처리를 아래와 같이 할수도 있을것 같습니다 😄 
```suggestion
        val inputValues = input?.split(" ") ?: throw IllegalArgumentException("입력값이 null 입니다.")
```